accession/admin_views.py

```diff
@@ -43,12 +43,12 @@ def print_view(request, app_label, model_name, object_id):
                     except Exception:
                         pass
                     else:
-                        value = entry.__unicode__().encode("utf-8")
+                        value = entry
```

Comment:
Since we are no longer doing the unicode thing here, maybe in line 40 we should change the variable name from `entry` to `value` and modify the comment above that to take out "unicode version"